### ISSUE TITLE
--env NODE_ENV is not working

### ISSUE THREAD

**Issue by ericntd** [Unitech/pm2#2142]
The commands: `pm2 start process.json --env production` and `pm2 start process.json --env random`

The results:
`console.log(process.env.NODE_ENV)` always gives **production**

My process.json:

``` javascript
{
    "apps" : [
        {
            "script"      : "server-local.js",
            "watch"       : ["views"],
            "ignore_watch": ["node_modules"],
            "env": {
                "NODE_ENV": "development",
            },
            "env_production" : {
                "NODE_ENV": "production"
            }
        }
    ]
}
```

The Node JS script:

``` javascript
console.log(process.env.NODE_ENV)
require('babel/register');
require('./express.js');
```

**Comment by Unitech** [comment]
Commands?

**Comment by dwelle** [comment]
I have it vice versa. It always starts in `development`, and ignores `env_production` when run with `--env=production`

**Comment by vmarchaud** [comment]
Because its not updated, it will be fixed when we publish `pm2 v2.3.x`
